src/setupProxy.js

```diff
@@ -0,0 +1,11 @@
+const { createProxyMiddleware } = require("http-proxy-middleware");
+
+module.exports = function (app) {
+  app.use(
+    "/backend",
+    createProxyMiddleware({
+      changeOrigin: true,
+      target: "https://cms.velocity-aachen.de/",
+    }),
+  );
+};
```

Comment:
The other redirects from the Netlify config could be configured here as well. I have not yet figured out how to do the rewrites necessary for the tile proxying.